config/routes.rb

```diff
@@ -160,15 +160,14 @@
   get "/dashboard/districts/", to: redirect("/reports/districts/")
   get "/dashboard/districts/:slug", to: redirect("/reports/districts/%{slug}")
   get "/reports/districts/", to: redirect("/reports/regions/")
-  get "/reports/districts/:slug", to: redirect("/reports/regions/facility_group-%{slug}")
+  # get "/reports/districts/:slug", to: redirect("/reports/regions/facility_group-%{slug}")
```

Comment:
@rsanheim is this intentionally commented out?

Reply:
oops, nope, ill remove it